app/client/components/alert/Alert.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropsWithChildren } from "react";
+import { AlertIcon } from "./AlertIcon";
+
+export type AlertType = "error" | "warning" | "info" | "success" | "debug";
+
+interface AlertProps {
+  type: AlertType;
+}
+
+const getAlertCSS = (type: AlertType) => {
+  switch (type) {
+    // All these class names need to be written as full strings,
+    // because Tailwind does not support dynamic class names.
```

Comment:
What about using `const enum AlertType` with string values? Const enum values are inlined during compilation, so I suppose Tailwind should be happy with them.

Reply:
I refactored it to an object.